### DISCUSSION TITLE
making it built-in vim

### DISCUSSION THREAD

**Discussion by jacknicklenson** [habamax/vim-dir#22] [General]
hi i love your contributions to vim eco system recently like comment and nohlsearch. My wish is can you make this plugin built-in to vim instead of netrw because netrw in windows copy is broken. it gives error and does not have a good UI/UX. However yours has so nice ui like emacs one. Can you make it built-in pls :)

**Comment by habamax** [comment]
It is not that easy :).

You will need to persuade vim core contributors to have additional file manager bundled with vim. And then other vocal vim users who might not like that idea.

**Comment by jacknicklenson** [comment]
oh, okay thanks anyway, your file manager 1000 times better than old buggy(copy command gives error on windows) netrw file manager. I wish it could be replaced it :/
